crates/bevy_pbr/src/render/parallax_mapping.wgsl

```diff
@@ -0,0 +1,106 @@
+#define_import_path bevy_pbr::parallax_mapping
+
+fn sample_depth_map(uv: vec2<f32>) -> f32 {
+    return textureSample(depth_map_texture, depth_map_sampler, uv).r;
```

Comment:
There were also log messages about using gradients for mip level calculating within a loop. We should probably calculate the uv gradient once for the base uv and then pass it in into this function and use… I think textureSampleGrad? The offset should not be toooo far from the original uv that a different mip levels is needed.